config.test.yaml

```diff
@@ -40,9 +35,8 @@ default_project: &default_project
         mobileapps:
           host: http://appservice.wmflabs.org
         events:
-          purge:
-            host: 127.0.0.1
-            port: 4321
+          uri: http://127.0.0.1:8085/v1/events
+          topic: resource_change
```

Comment:
We didn't add the events config here before, because
- it's not strictly needed for testing
- not adding it verified that RB doesn't crash if the config is not there
- it makes tests slower a little bit since we make some additional requests

I don't have a strong opinion whether we should remove these 2 lines or leave them, up to you